packages/react/src/internal/components/InputLabel.tsx

```diff
@@ -54,7 +54,6 @@ const InputLabel: React.FC<React.PropsWithChildren<Props>> = ({
         display: 'block',
         color: disabled ? 'fg.muted' : 'fg.default',
         cursor: disabled ? 'not-allowed' : 'pointer',
-        alignSelf: 'flex-start',
```

Comment:
Just making sure this is an intended change? as it doesn't seem directly correlated with Checkbox